core/src/main/java/nu/yona/server/exceptions/ResourceBasedException.java

```diff
@@ -100,11 +80,6 @@ protected ResourceBasedException(HttpStatus statusCode, Throwable t, String mess
 		this.statusCode = statusCode;
 	}
 
-	public static void setTranslator(Translator translator)

```

Comment:
Actually, this public method is a better injection than having an inline `Translator.getInstance()`. For unit testing, it is otherwise impossible to change the dependency.

Reply:
We currently don't have tests that depend on this, but if we want to do it, we can use a similar setter in ``Translator``. The code to get the instance in a static context needs to be in ``Translator``, as that's the only one to logically depend upon.